admin-dev/themes/new-theme/js/app/utils/fields.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+const $ = global.$;
+
+/**
+ * Enable all datepickers.
+ */
+const initInvalidFields = () => {
+  $('input,select,textarea').each(function findElements() {
+    $(this).on('invalid', function scroll() {
```

Comment:
You should use a [delegated event](https://learn.jquery.com/events/event-delegation/) instead of adding one event handler for each element.

Reply:
I tried, but it's not working with `invalid` event.
Tests with `window` and `document` but it's not working.
https://github.com/PrestaShop/PrestaShop/pull/13219#discussion_r272651959

Reply:
But I refactor my code :)